src/main.rs

```diff
@@ -90,8 +90,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .unwrap();
 
     if matches.is_present("wait") {
-        info!("Waiting 90 seconds");
-        sleep(Duration::from_secs(90));
+        info!("Waiting 5 seconds");
```

Comment:
I figure a short delay might be useful? I tried this to see if the is_connected() function would work again. No change, is_connected() is still broke.